abbvie-external/OmicNavigatorWebApp
# [IPAD-382] Plotly export

Problem: For our PNG/SVG export, the "Expression Heatmap interactive..." plot **iheatmapr** doesn't export labels/legend, while **heatmaply** and **plotly** do. Plotly's PNG/SVG export function does export **iheatmapr** labels/legend just fine. 

Solution: Hide the plotly export button (using CSS rather than in the plotly config) but initiate a click on it for Plotly exports, after updating export type (png, svg).

To Test: 
1) Export the both plot types (non-plotly and plotly - the problematic iheatmapr), in both formats (svg, png), in four export areas (differential dynamic single-feature tab, differential dynamic multi-feature tab, differential static overlay (single-feature and multi-feature), and enrichment triple pane (only supports single-feature)
2) Confirm the charts look correct (iheatmapr has labels/legends)
3) Confirm the export name makes sense (no undefined)
